jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -133,6 +141,37 @@ private static void initEncodings() {
 			osEncoding = definedOSEncoding;
 	}	
 		
+	/*[IF Sidecar18-SE-OpenJ9 | JAVA_SPEC_VERSION >= 11]*/
+	/*
+	 * Return the Charset name of the primary or fallback encoding, if different from the default.
+	 */
+	static String getCharsetName(String primary, String fallback, Charset defaultCharset) {
+		if (primary != null) {
+			try {
+				Charset newCharset = Charset.forName(primary);
+				if (newCharset.equals(defaultCharset)) {
+					return null;
+				} else {
+					return newCharset.name();
+				}
+			} catch (IllegalCharsetNameException | UnsupportedCharsetException e) {
+			}
```

Comment:
An explicit comment explaining why the exceptions are being ignore would be welcome.